# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/qt5base/qfloat16.patch
# Copyright (C) 2020 - 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- qtbase/src/corelib/global/qfloat16.cpp.vanilla	2020-05-17 21:58:04.029166720 +0200
+++ qtbase/src/corelib/global/qfloat16.cpp	2020-05-17 22:03:11.526165126 +0200
@@ -282,5 +282,5 @@
 
 #include "qfloat16tables.cpp"
 #ifdef QFLOAT16_INCLUDE_FAST
-#  include "qfloat16_f16c.c"
+//#  include "qfloat16_f16c.c"
 #endif
